SwiftWisdom/Core/Foundation/Data/DataConvertible.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  DataConvertible.swift
+//  SwiftWisdom
+//
+//  Created by Paul Rolfe on 2/3/17.
+//  Copyright © 2017 Intrepid. All rights reserved.
+//
+
```

Comment:
I'll add a note that I borrowed this protocol from http://stackoverflow.com/questions/38023838/round-trip-swift-number-types-to-from-data